### ISSUE TITLE
Publish Keras Weights

### ISSUE THREAD

**Issue by ashvinnihalani** [chandrikadeb7/Face-Mask-Detection#133]
## 🚀 Feature

Publish weights in a .h5 file similar to how other popular Keras models do in the release section

## Motivation

Enable other projects to use the model in machine learning applications. 

## Pitch

Use the existing Keras [functionality](https://www.tensorflow.org/guide/keras/save_and_serialize) to post the weights to the release section of the Github page. Initially, it can be done manually but eventually, we can set up CI to do it on every merge to master/release.
